tendrl/node_agent/node_sync/sds_detect.py

```diff
@@ -42,39 +42,39 @@ def sync():
                         integration_index_key = \
                             "indexes/detected_cluster_id_to_integration_id/" \
                             "%s" % sds_details['detected_cluster_id']
-
-                        if "provisioner/%s" % NS.tendrl_context.integration_id \
-                            in NS.node_context.tags:
+                        _ptag = "provisioner/%s" % \
+                            NS.tendrl_context.integration_id
+                        if _ptag in NS.node_context.tags:
```

Comment:
@GowthamShanmugam is the issue mentioned around this condition resolved? Do we need some fix here?